homeassistant/components/lutron/__init__.py

```diff
@@ -53,7 +54,31 @@
 )
 
 
-def setup(hass: HomeAssistant, base_config: ConfigType) -> bool:
+async def async_setup(hass: HomeAssistant, base_config: ConfigType) -> bool:
+    """Set up the Lutron component."""
+    hass.data.setdefault(DOMAIN, {})
```

Comment:
This can be removed or should be placed in the setup entry

Reply:
agree and done